integration/kubernetes/k8s-job.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	get_pod_config_dir
+}
+
+@test "Run a job to completion" {
+	job_name="job-pi-test"
+	wait_time=20
+	sleep_time=2
+
+	# Create job
+	kubectl apply -f "${pod_config_dir}/job.yaml"
+
+	# Verify job
+	kubectl describe jobs/"$job_name" | grep SuccessfulCreate
+
+	# List pods that belong to the job
+	pod_name=$(kubectl get pods --selector=job-name="$job_name" --output=jsonpath='{.items[*].metadata.name}')
+
+	# Check job was completed
+	cmd="kubectl get pods -o wide | grep Completed"
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
```

Comment:
Similarly, I'd probably prefer a `kubectl wait` here, which can also take a timeout param. wdyt?

Reply:
@GabyCT What about the above suggestion from @grahamwhaley.
Why not use wait instead:
```
 kubectl wait --for=condition=complete --timeout=32s job/worker
```